### ISSUE TITLE
Invalid element templates are still offered for application

### ISSUE THREAD

**Issue by MaxTru** [camunda/camunda-modeler#2110]
__Describe the Bug__

Even though an error is found in an element template (in https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/lib/provider/camunda/element-templates/Validator.js), the template will be offered for application in the element templates modal.

__Steps to Reproduce__

1. Setup invalid local template
```json
[
  {
    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.2.0/resources/schema.json",
    "name": "Default Rendering2",
    "id": "com.camunda.example.DefaultRendering2",
    "appliesTo": [
      "bpmn:ServiceTask"
    ],
    "properties": [
      {
        "label": "Input Parameter",
        "binding": {
          "type": "camunda:inputParameter",
          "name": "processVar",
          "asdf": "aasdasdasdf"
        }
      },
      {
        "label": "Output Parameter",
        "type": "I dont exist",
        "binding": {
          "type": "camunda:outputParameter",
          "source": "outputParam"
        }
      }
    ]
  }]
```

Note that `"type": "I dont exist",` is invalid
2. Start Camunda Modeler
3. Camunda Modeler will log:
```
[diagram_1.bpmn] invalid property type <I dont exist>; must be any of { String, Text, Boolean, Hidden, Dropdown }
    at Validator._logError (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:249:13)
    at Validator._validateProperty (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:191:18)
    at Array.filter (<anonymous>)
    at Validator._validateProperties (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:171:38)
    at Validator._validateTemplate (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:124:17)
    at Validator.add (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:59:20)
    at Array.forEach (<anonymous>)
    at Validator.addAll (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:44:17)
    at ElementTemplatesLoader.setTemplates (webpack-internal:///../../../bpmn-io/bpmn-js-properties-panel/lib/provider/camunda/element-templates/ElementTemplatesLoader.js:59:35)
    at BpmnEditor.loadTemplates (webpack-internal:///./src/app/tabs/bpmn/BpmnEditor.js:419:21) [ error ]
```
4. Click on Service Task and open Element Templates Modal
5. => Element Template is still offered as valid option.

__Expected Behavior__

I cannot apply invalid element templates

__Environment__

- OS: Arch Linux
- Camunda Modeler Version: 4.6

**Comment by pinussilvestrus** [comment]
Seems like [we don't apply the validator](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/element-templates-modal/components/ElementTemplatesModalView.js#L61) inside the Catalog Modal.

**Comment by pinussilvestrus** [comment]
Closed via https://github.com/camunda/camunda-modeler/commit/bffaf7a074bf44d3eed66d36bb4a7721ef2aa5cb

**Comment by pinussilvestrus** [comment]
Why only fixed upstream? The fixed was actually inside this project (https://github.com/camunda/camunda-modeler/commit/bffaf7a074bf44d3eed66d36bb4a7721ef2aa5cb)

**Comment by MaxTru** [comment]
> Why only fixed upstream? The fix was actually inside this project ([bffaf7a](https://github.com/camunda/camunda-modeler/commit/bffaf7a074bf44d3eed66d36bb4a7721ef2aa5cb))

Sorry, true. My mistake
